### PR TITLE
Unification work + bye KubeJS ingots

### PR DESCRIPTION
**Pull request by WithersChat**
Basically adding textures to Omnium, Neutronium, Infinity and the like, directly on the GT items. Might take some time, drafting this so nobody does duplicate work.

### CODE REVIEW

**Comment by WithersChat** [comment]
Ready for merging

File: kubejs/startup_scripts/Material_Registry.js [hunk 1]
```diff
@@ -1,16 +1,17 @@
 // Elements
 GTCEuStartupEvents.registry('gtceu:element', event => {
+    event.create('infinity', 69, 420, -1, null, '∞', false); //feel free to add a better proton/neutron count
```
Comment (LetterN) [inline kubejs/startup_scripts/Material_Registry.js:3]:
```suggestion
```
dont make an **element** since nobody "inherits" infinity ingot thru `.components()`

File: kubejs/startup_scripts/Material_Registry.js [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
     event.create('draconium_awakened', 149, 264, -1, null, 'Dc*', false)
     event.create('taranium', 149, 264, -1, null, 'Tn', false)
 })
 
-GTCEuStartupEvents.materialModification(() => {
-	GTMaterials.get('infinity').setFormula("∞");
+// Elemental materials
+GTCEuStartupEvents.registry('gtceu:material_icon_set', event => {
+    event.create('omnium').parent(GTMaterialIconSet.SHINY)
 })
```
Comment (LetterN) [inline kubejs/startup_scripts/Material_Registry.js:13]:
```suggestion
GTCEuStartupEvents.materialModification(() => {
	GTMaterials.get('infinity').setFormula("∞");
})
// Elemental materials
GTCEuStartupEvents.registry('gtceu:material_icon_set', event => {
    event.create('omnium').parent(GTMaterialIconSet.SHINY)
})
```

File: kubejs/startup_scripts/Material_Registry.js [hunk 3]
```diff
@@ -401,14 +407,14 @@ GTCEuStartupEvents.registry('gtceu:material', event => {
...
-		.color(0x000000)
-		.iconSet("shiny")
-		.flags(GTMaterialFlags.GENERATE_PLATE)
-		.components('5x neutronium');
+    event.create('infinity')
+        .ingot()
+        .element(GTElements.get("infinity"))
+        .color(0xffffff)
+        .iconSet('infinity')
+        .flags(GTMaterialFlags.GENERATE_PLATE)
```
Comment (LetterN) [inline kubejs/startup_scripts/Material_Registry.js:417]:
```suggestion
    event.create('infinity')
        .ingot()
        .element(GTElements.get("infinity"))
        .color(0xffffff)
        .iconSet('infinity')
        .flags(GTMaterialFlags.GENERATE_PLATE)
        .components('5x neutronium');
```
why did you remove the components? it is intentional that this can be decomposed to 5 neutronium

**Review by LetterN — CHANGES_REQUESTED** [review]

File: kubejs/startup_scripts/Material_Registry.js (hunk 1 above)
Comment (WithersChat) [reply kubejs/startup_scripts/Material_Registry.js:3]: Why does it matter tho? Nobody inherits omnium either, and it got an element anyways.

File: kubejs/startup_scripts/Material_Registry.js (hunk 3 above)
Comment (WithersChat) [reply kubejs/startup_scripts/Material_Registry.js:417]: ...why?

Comment (WithersChat) [reply kubejs/startup_scripts/Material_Registry.js:417]: Marked as resolved because the decomposition is useless (infinity requires almost 2 stacks of neutronium per)

File: kubejs/startup_scripts/Material_Registry.js (hunk 1 above)
Comment (WithersChat) [reply kubejs/startup_scripts/Material_Registry.js:3]: Marked as resolved because all single-element materials have their element, even if there isn't any alloy using them (souce: modpack made by GTM dev)

File: kubejs/startup_scripts/Material_Registry.js (hunk 2 above)
Comment (WithersChat) [reply kubejs/startup_scripts/Material_Registry.js:13]: Marked as resolved because Infinity has an element
